initfiles/sbin/prerm.in

```diff
@@ -28,11 +28,12 @@ fi
 # Stop all services before cleaning up
 ######################################
 if which systemctl 1>/dev/null 2>&1; then
-	systemctl stop hpccsystems-platform.target
+    systemctl stop hpccsystems-platform.target
 else 
 	/etc/init.d/hpcc-init stop
 	/etc/init.d/dafilesrv stop
 fi
+killall -9 -u hpcc
```

Comment:
Shouldn't the user name be configurable?

But also a bit concerned about the situation where it is - on local installs I run the platform as myself, but I wouldn't want all processes owned by myself killed on removal. Maybe this code is not executed in such cases though

Reply:
The code in the prerm script is not used for 'developer builds' where we change the runtime user and installation location.  That's why file paths and username in this file are hardcoded and it's never been a problem we had to go back and fix. This file only gets embedded in the deb and rpm packages.

Reply:
The code in install-init.in seems to have user configurable...

Reply:
I'll go ahead and make changes to the uninstall-init, postinst, postrm, and prerm scripts and make sure they work the same way as the install-init script.